packages/bpk-component-icon/src/withDescription.js

```diff
@@ -25,14 +25,19 @@ import STYLES from './withDescription.scss';
 
 const getClassName = cssModules(STYLES);
 
-export default function withDescription(Component: ComponentType<any>, description: string): ComponentType<any> {
+export default function withDescription(
+  Component: ComponentType<any>,
+  description: string,
+): ComponentType<any> {
   const WithDescription = props => {
     const { children, ...rest } = props;
 
     return (
       <span>
         <Component {...rest}>{children}</Component>
-        <span className={getClassName('bpk-icon-description')}>{description}</span>
+        <span className={getClassName('bpk-icon-description')}>
+          {description}
+        </span>
       </span>
     );
   };
```

Comment:
This file isn't related to this PR so shouldn't have changed - could you revert the changes here?